src/app/embed/embed.component.ts

```diff
@@ -15,11 +16,13 @@ const PYM_CHILD_ID_PARAM = 'childId';
   styleUrls: ['embed.component.css'],
   providers: [MergeAdapter, QSDAdapter, DraperAdapter, FeedAdapter],
   template: `
-    <play-share-modal *ngIf="showShareModal" (close)="hideModal()">
-    </play-share-modal>
+    <play-share-modal *ngIf="showShareModal" (close)="hideModal()"></play-share-modal>
     <play-player [feedArtworkUrl]="feedArtworkUrl" [audioUrl]="audioUrl" [title]="title" [subtitle]="subtitle"
       [subscribeUrl]="subscribeUrl" [subscribeTarget]="subscribeTarget" [artworkUrl]="artworkUrl" (share)="showModal()"
       [showPlaylist]="showPlaylist" [episodes]="episodes">
+      <ng-template let-dismiss="dismiss">
+        <p>Placeholder for overlay content (could put share modal here if we wanted)</p>
```

Comment:
So the idea is we'll use this in place of the share modal, in some future PR then?  That works for me.  More consistent UX!

Reply:
yes